src/main/java/com/faforever/client/coop/CoopController.java

```diff
@@ -194,6 +201,24 @@ public void initialize() {
     });
   }
 
+  private Predicate<CoopResultBean> onSearchCreatePredict(String newValue){
+    leaderboardTable.setItems(observableList(leaderboardFilteredList));
```

Comment:
This should be moved into the initialize function.

You also don't need to wrap the filtered list in an observablelist